test/functional/direct_connection.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const expect = require('chai').expect;
+const MongoClient = require('../../lib/mongo_client.js');
+
+describe('directConnection uri error', function() {
```

Comment:
I wonder if this is the best place for these tests. They all seem to expect an error, but don't specify what the error actually is. Since this is testing it in a "functional" way (the driver is actually trying to connect to an SRV connection string here). Are these prose tests for `mongodb+srv` parsing?

Reply:
I can remove this whole test file.

It was more or a guide to understand the discrepancies between casing, and the different combination for connection string and options.

Reply:
I canned it 🗑